simcon_project/simcon_project/settings.py

```diff
@@ -138,7 +138,9 @@
 
 # Static URLS [These are used when in non production enviroments]
 STATIC_URL = '/static/'
-MEDIA_URL = '/media/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
```

Comment:
i think our media folder is inside our static folder

Reply:
Hmm I didn't see anything in there. I remember when we were mobbing that someone created a media folder at project-level but we must not have kept it around.